.github/workflows/upstreamed.yaml

```diff
@@ -20,12 +21,34 @@ jobs:
   docs:
     name: Publish Documentation
     runs-on: ubuntu-20.04
+    continue-on-error: false
+    if: github.repository == 'fedora-modularity/libmodulemd'
     steps:
-      - name: Checkout code
+      - name: Checkout code repo
         uses: actions/checkout@v2
 
-      - name: Publish
-        env:
-          DOC_TOKEN: ${{ secrets.DOC_TOKEN }}
+      - name: Checkout documentation repo
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.actor }}/fedora-modularity.github.io
```

Comment:
Will this work when run by anyone except `sgallagher`? (Unless the person triggering this also has an appropriate fork and updates `secrets.DOC_TOKEN` to match. At least that's my understanding of how this works.)

Perhaps there should also be a `secrets.DOC_USER` that pairs with `secrets.DOC_TOKEN` that gets used here? (I can't seem to find a way to set per-repository non-secret variables other than hard coding them in the workflow file.)

Reply:
Oops, that was testing code. That should just say `fedora-modularity`. I was using my own fork to test and that was an easy way to do it...